docs/docs/development/docs.md

```diff
@@ -5,19 +5,115 @@
 
 # Documentation
 
-Write documentation if necessary so that developer, administrators and users
-have an easily accessible way of learning about the collaboration manager.
+It's important to have a good and up to date documentation for the
+Collaboration Manager. Documentation is an accessible way to share knowledge
+for developers, administrators and users. It can also be used by support teams
+to resolve issues more efficiently.
 
-## Setup and Serve the Documentation
+## Development Setup
 
 Navigate to the `docs` folder. With the commands `make install` and
 `make serve` you can build the documentation and find it served under
-`http://127.0.0.1:8081/`. Here you can try your newly written documentation or
-search for something yourself.
+`http://127.0.0.1:8081/`. It's a live-preview of the current version and does
+automatically refresh on file-changes.
```

Comment:
Maybe a little more specific/clear:

Navigate to the `docs` folder from the command line. Executing the command `make install` followed by `make serve` will build and host the documentation for viewing locally in a browser at `http://127.0.0.1:8081/`. This live preview of the documentation automatically refreshes to reflect any changes made to its source files until the `make serve` process is terminated.